.nais/alerts.yaml

```diff
@@ -23,17 +23,31 @@ spec:
             alert_type: custom
             severity: critical
 
-        - alert: Fyllut er nede
+        - alert: Fyllut is unavailable
           expr: kube_deployment_status_replicas_available{deployment="skjemautfylling"} == 0
           for: 1m
           annotations:
-            title: "Fyllut er nede"
-            consequence: "Tjenesten er utilgjengelig for brukere. Undersøk hvorfor poddene er nede."
+            title: "Fyllut is unavailable"
+            consequence: "Service is unavailable to users. "
             action: {{logs-url}}
             sla: respond within 1h, during office hours
           labels:
             service: Fyllut
             namespace: fyllut-sendinn
-            special_type_to_use_in_alertmanager_config: "fyllut-sendinn-alerts"
+            special_type_to_use_in_alertmanager_config: {{special-type-to-use-in-alertmanager-config}}
+            alert_type: custom
+            severity: critical
+
+        - alert: Request to Exstream pdf failed
+          expr: (sum(max_over_time(fyllut_exstream_pdf_failures_total{}[3m]) or vector(0)) - sum(max_over_time(fyllut_exstream_pdf_failures_total{}[3m] offset 3m) or vector(0))) > 0
```

Comment:
Kort forklaring til expr: Jeg kunne ikke bruke `increase` pga at metric'en ikke finnes etter pod-restart før første failure faktisk har oppstått. Fant en workaround på nett hvor man med `max_over_time`, default til `vector(0)` i de tilfellene den ikke eksisterer, og bruk av `offset 3m` for å sjekke antall feil nå opp mot hva antallet var for 3 minutter siden.

En svakhet som jeg tror expr har er dersom en request til exstream pdf feiler like etter pod restart, for da kan antall feil før restart være høy, f.eks. 5, og antall feil like etter restart vil være 1, og expr vil da evaluere til false, og vi blir ikke varslet. Men i og med at `for: 30s` så vil dette bare kunne oppstå hvis request feiler de første 30 sekundene +/- etter restart hvis jeg har forstått det riktig.

Et alternativ er å initialisere counter med alle mulige kombinasjoner av labels (formPath og submissionMethod), og sørge for at antall settes til 0 før pod'en scrapes for metrics. Det kan vi jo se på etterhvert kanskje.